src/ducks/apps/index.js

```diff
@@ -149,25 +149,32 @@ function _sortAlphabetically (array, property) {
   return array.sort((a, b) => a[property] > b[property])
 }
 
+/* Only for the icon fetching */
+const root = document.querySelector('[role=application]')
+const data = root && root.dataset
+const COZY_TOKEN = data && data.cozyToken
+const COZY_DOMAIN = data && (`//${data.cozyDomain}`)
+/* Only for the icon fetching */
+
 async function _getIcon (url) {
   if (!url) return ''
   let icon
   try {
-    icon = await cozy.client.fetchJSON('GET', url)
+    const resp = await fetch(`${COZY_DOMAIN}${url}`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        Authorization: `Bearer ${COZY_TOKEN}`
+      }
+    })
+    if (!resp.ok) throw new Error (`Error while fetching icon: ${resp.statusText}: ${url}`)
```

Comment:
I think we don't need to throw an error here if we simply ignore it two lines below. Am I right ?

Reply:
It could be useful for debugging and it was to force using the catch statement here

Reply:
I will remove this throw statement in my next PR ;)